docs/tutorials/semantic_search/asymmetric_embedding_model.md

```diff
@@ -0,0 +1,541 @@
+# Tutorial: Running Asymmetric Semnantic Search within OpenSearch
+
+This tutorial demonstrates how generating text embeddings using an asymmetric embedding model in OpenSearch. The embeddings will be used
+to run semantic search, implemented using a Docker container. The example model used in this tutorial is the multilingual
+`intfloat/multilingual-e5-small` model from Hugging Face. 
+You will learn how to prepare the model, register it in OpenSearch, and run inference to generate embeddings.
+
+> **Note**: Make sure to replace all placeholders (e.g., `your_`) with your specific values.
+
+---
+
+## Prerequisites
+
+- Docker Desktop installed and running on your local machine.
```

Comment:
IMO this can be confusing to customers. Why docker desktop installation needs to be a Prerequisite to run asymmetric embedding model.

Reply:
Prerequisites can be a running OS cluster and then may be add a link how to setup OS cluster if you want.